packages/site-server/src/catalog/router.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import Router from '@koa/router';
+
+export const catalogRouter = new Router();
+
+catalogRouter.get('/element/:path+', async (context) => {
+  const {params} = context;
+
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const elementPath = params['path']!;
+  const elementPathSegments = elementPath.split('/');
+  const isScoped = elementPathSegments[0]?.startsWith('@');
+  const packageName = isScoped
+    ? elementPathSegments[0] + '/' + elementPathSegments[1]
+    : elementPathSegments[0]!; // eslint-disable-line @typescript-eslint/no-non-null-assertion
+  const elementName = elementPathSegments[isScoped ? 2 : 1];
+  
+  context.body = `
```

Comment:
Presumably we would factor the actual route handler for each page to another module for organization? So that router.js only does routing? Fine for now though.

Reply:
Yeah, the next PR moves this to a template, but puts a GraphQL call right here. Was planning on refactoring to more modules over time.